### ISSUE TITLE
bidiphase UnboundLocalError

### ISSUE THREAD

**Issue by willyh101** [MouseLand/suite2p#856]
Registration fails with `'do_bidiphase': True`. Running the most recent pip installed version from GitHub (fetched today).

```
Traceback (most recent call last):
  File "run_s2p_8m.py", line 107, in <module>
    process_data(ops=default_ops, db=db)
  File "run_s2p_8m.py", line 90, in process_data
    run_s2p(ops=ops,db=db)
  File "c:\users\will\code\suite2p\suite2p\run_s2p.py", line 429, in run_s2p
    op = run_plane(op, ops_path=ops_path)
  File "c:\users\will\code\suite2p\suite2p\run_s2p.py", line 292, in run_plane
    ops = pipeline(f_reg, f_raw, f_reg_chan2, f_raw_chan2, run_registration, ops, stat=stat)
  File "c:\users\will\code\suite2p\suite2p\run_s2p.py", line 78, in pipeline
    registration_outputs = registration.register.registration_wrapper(f_reg, f_raw=f_raw, f_reg_chan2=f_reg_chan2, f_raw_chan2=f_raw_chan2,
  File "c:\users\will\code\suite2p\suite2p\registration\register.py", line 560, in registration_wrapper
    outputs = compute_reference_and_register_frames(f_align_in, f_align_out=f_align_out, refImg=refImg, ops=ops)
  File "c:\users\will\code\suite2p\suite2p\registration\register.py", line 373, in compute_reference_and_register_frames
    bidiphase = bidiphase.compute(frames)
UnboundLocalError: local variable 'bidiphase' referenced before assignment
```

**Comment by willyh101** [comment]
fixed in #857

**Comment by chriski777** [comment]
Merged, closing now. Thanks, Will!
